resources/lang/fr/downloads.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 return [
-    'none' => 'Aucun téléchargement!',
```

Comment:
I think "Aucun téléchargements!" (No downloads) is better than "Rien..." (Nothing)

Reply:
I don't know, anyway if you prefer "Aucun téléchargements!", then I would take of the last S, "Aucun téléchargement!"
But I would prefer "Rien à télécharger!"

Reply:
"Rien à télécharger!" is good for me. "rien..." was not enough